src/Forms/_FormValidationOverlay.test.js

```diff
@@ -0,0 +1,137 @@
+import { act } from 'react-dom/test-utils';
+import FormValidationOverlay from './_FormValidationOverlay';
+import { mount } from 'enzyme';
+import React from 'react';
+
+describe('<FormValidationOverlay />', () => {
+    const setup = (props = {}) => mount(
+        <FormValidationOverlay control={(<input onChange={() => {}} value='Test' />)} {...props} />
+    );
+
+    afterEach(() => {
+        document.body.innerHTML = '';
+    });
+
+    const getFormMessage = () => document.body.querySelector('.fd-popover__popper > div > .fd-form-message');
```

Comment:
Just an idea or an alternative, I've seen tests that include
```jsx
// Ensure the popper is accessible with enzyme
jest.mock('react-dom', () => {
    const original = jest.requireActual('react-dom');
    return {
        ...original,
        createPortal: (node) => node,
    };
});
```
so you can still access the elements via `enzyme`

Reply:
I like this, but I'd like to apply it globally in `config/jest/setup.js` so we don't have some unit tests with it/some without. It makes 450 unit tests fail at the moment so I'll probably skip it on this pr 😆